config/packages/doctrine.yaml

```diff
@@ -1,6 +1,7 @@
 doctrine:
     dbal:
         url: '%env(resolve:DATABASE_URL)%'
+        use_savepoints: true
```

Comment:
You mention we enable this as our db engine supports it. But I do not see what change of behavior this brings? 

Might be worthwhile to describe this is the default behavior that is made explicit here? And that it ensures nested transactions are handled in a certain manner?

Reply:
Thanks, I clarified the commit message!